coordinator.go

```diff
@@ -117,7 +121,8 @@ func newCoordinator(schemas []*tableSchema, client *spanner.Client) *coordinator
 				tableName: schema.tableName,
 				client:    client,
 			},
-			referencedBy: []*table{},
+			referencedBy:   []*table{},
+			hasGlobalIndex: false,
```

Comment:
nit: In Go, using zero value is encouraged, so we don't need to write `hasGlobalIndex: false` here.

Reply:
removed `hasGlobalIndex: false`